test/integration/test_summarization_accuracy.py

```diff
@@ -42,7 +42,7 @@ def test_evaluate_sample(self, integration_tests_dir):
                 elif eval_score.name == ROUGE_SCORE:
                     assert eval_score.value == approx(0.250, abs=ABS_TOL)
                 elif eval_score.name == BERT_SCORE:
-                    assert eval_score.value == approx(0.734, abs=ABS_TOL)
+                    assert eval_score.value == approx(0.748, abs=ABS_TOL)
```

Comment:
why?

Reply:
Upgrading transformers version to ^4.36.0 caused the bertscore values to be slightly different. 

I did some testing and found that bertscore output differs for < v4.24.0 and >= v4.24.0, (previously we were using v4.22.1). This issue is similar to what was observed in this [github issue](https://github.com/Tiiiger/bert_score/issues/164). 

I wasn't able to find the root cause for the change but it seems like this occurs sometimes from pytorch/transformers upgrades, see [previous issue](https://github.com/Tiiiger/bert_score/issues/147).